ravenframework/CodeInterfaceBaseClass.py

```diff
@@ -115,20 +117,20 @@ def genCommand(self, inputFiles, executable, flags=None, fileArgs=None, preExec=
     returnCommand = subcodeCommand,outputfileroot
     return returnCommand
 
-  def readMoreXML(self, xmlNode, ravenWorkingDir):
+  def readXML(self, xmlNode, **kwargs):
```

Comment:
Is there a good reason for switching this to a **kwargs?  Also, the docstring has not been updated.

Reply:
See above comment, I'm trying to avoid overwriting this method. I can change it to workingDir = ravenWorkingDir. How do you think?

Reply:
Updated.

Reply:
Thanks.